### PR TITLE
Profiles for TPV and Dune HD homatics. New Apps 1und1 TV and Arte and special Apps for TPV (Philips) TV

### PR DESCRIPTION
**Pull request by Kat-CeDe**
Profiles for the devices Dune HD Homatics Box R 4K Plus and TPV (Philips) AndroidTV TVs
In my eyes the special Apps for TPV TV are the most interesting changes.

### CODE REVIEW

File: intg-androidtv/apps.py
Comment (zehnm) [inline intg-androidtv/apps.py:1]: Linting issue: https://github.com/unfoldedcircle/integration-androidtv/pull/43/checks

Probably missing comma in line 80

**Review by zehnm — CHANGES_REQUESTED** [review]
Cool, thanks for the additional profiles!
Please fix the linting issue and the CHANGELOG, then I'll merge it and create a new release.

File: intg-androidtv/apps.py
Comment (zehnm) [reply intg-androidtv/apps.py:1]: There's still an issue: 

https://github.com/unfoldedcircle/integration-androidtv/actions/runs/8874808081/job/24362930301?pr=43#step:7:12

You can check & format the code locally with Black: https://github.com/unfoldedcircle/integration-androidtv/blob/main/docs/code_guidelines.md#format-code

File: intg-androidtv/apps.py [hunk 1]
```diff
@@ -22,7 +22,16 @@
...
+    "1und1": {"url": "1und1tv://1und1.tv"},
+    "Arte": {"url": "arte://display"},
+    "Google Play Store": {"url": "https://play.google.com/store/"},
+    "DVB-C/T/S": {"url": "market://launch?id=org.droidtv.playtv"},
+    "ATV Inputs": {"url": "market://launch?id=org.droidtv.channels"},
+    "ATV PlayFI": {"url": "market://launch?id=com.phorus.playfi.tv"},
+    "ATV Now on TV": {"url": "market://launch?id=org.droidtv.nettvrecommender"},
+    "ATV Media": {"url": "market://launch?id=org.droidtv.contentexplorer"},
+    "ATV Browser": {"url": "market://launch?id=com.vewd.core.browserui"},
+ }
```
Comment (zehnm) [inline intg-androidtv/apps.py:34]: Linting issue https://github.com/unfoldedcircle/integration-androidtv/actions/runs/8891314506/job/24413032213?pr=43

You can check & format the code locally with Black: https://github.com/unfoldedcircle/integration-androidtv/blob/main/docs/code_guidelines.md#format-code

Most IDEs allow to configure "save-actions" to always format the code when it's saved.

**Review by zehnm — CHANGES_REQUESTED** [review]

**Comment by Kat-CeDe** [comment]
I tried to add black code to GitHub and got the recommendation to add this. Before I used black code on windows with VS code but this formated completely different than yours.

Ralf Biedermann

----
Die Lücke, die ich hinterlasse, ersetzt mich vollständig!

RBsoft
----
----------------------------------------

30.04.2024 13:18:33 Markus Zehnder ***@***.***>:

**Comment by zehnm** [comment]
> I tried to add black code to GitHub and got the recommendation to add this.

Please restore `.github/ISSUE_TEMPLATE/config.yml` in the pull request.
If you enable GitHub actions in your repository, then the same checks are run on each push.

> Before I used black code on windows with VS code but this formated completely different than yours.

You can use the command line as described here: https://github.com/unfoldedcircle/integration-androidtv/blob/main/docs/code_guidelines.md#tooling

**Comment by zehnm** [comment]
Please fix the linting issue if you get a chance.
All checks must be successful to merge a PR.

**Review by zehnm — APPROVED** [review]
All good now and thanks again for the PR!
I'll create a new release soon and include it in the next beta firmware.

**Pull request merged**